crates/kitsune_p2p/types/src/config.rs

```diff
@@ -435,22 +414,41 @@ pub struct KitsuneP2pConfig {
     #[serde(default)]
     pub tuning_params: KitsuneP2pTuningParams,
 
-    /// The network used for connecting to other peers
-    pub network_type: NetworkType,
-
     /// All tracing logs from kitsune tasks will be instrumented to contain this string,
     /// so that logs from multiple instances in the same process can be disambiguated.
     #[serde(default)]
     pub tracing_scope: Option<String>,
 }
 
+#[cfg(feature = "test_utils")]
 impl Default for KitsuneP2pConfig {
     fn default() -> Self {
+        Self::mem()
+    }
+}
+
+impl KitsuneP2pConfig {
+    /// Minimal but non-functional config. Without a transport pool set,
+    /// nothing will work.
```

Comment:
This has the in-memory transport pool set. Does that mean this transport pool is a dummy?

Reply:
I fixed this comment. I missed it when reverting other things.